Makefile

```diff
@@ -140,12 +141,19 @@ test-e2e: ## Run end-to-end tests using an available cluster.
 ##@ Release Artifacts
 ######################
 
-cross-platform: darwin linux windows ## Build all distributable (cross-platform) binaries
+cross-platform: darwin-arm64 darwin-amd64 linux windows ## Build all distributable (cross-platform) binaries
 
-darwin: $(BIN_DARWIN) ## Build for Darwin (macOS)
+darwin-arm64: $(BIN_DARWIN_ARM64) ## Build for mac M1
+	env CGO_ENABLED=0 GOOS=darwin GOARCH=arm64 go build -o $(BIN_DARWIN_ARM64) -ldflags $(LDFLAGS) ./cmd/$(BIN)
```

Comment:
Here's one line to remove.  This make target is just an alias and need not have contents